Bio/SeqUtils/__init__.py

```diff
@@ -11,42 +11,111 @@
 
 
 import re
+import warnings
+from collections import Counter
 from math import pi, sin, cos
 
 from Bio.Seq import Seq
 from Bio.Data import IUPACData
-
+from Bio import BiopythonDeprecationWarning
 
 ######################################
 # DNA
 ######################
 # {{{
 
+# Nucleotides not present here are treated as having value 0.
```

Comment:
That worries me, especially if you think the dictionary is worth adding as a public variable. I would be much happier with the A, T, and S explicitly included as zero.

Reply:
I believe it's better to keep it private then. But even with that I can include the A T and S values if you believe it would improve clarity.